src/main/resources/webinterface/css/simple-sidebar.css

```diff
@@ -113,4 +113,66 @@
     #wrapper.toggled #sidebar-wrapper-dwarfs {
         width: 250px;
     }
+}
+
+/* Sidebar Icons */
+#sidebar-wrapper-elves .carrotAction {
+	background-image: url("../icons/carrot.png");
+	background-repeat: no-repeat;
+	background-position: 10%;
```

Comment:
Since each element has the following constants:

- background-repeat
- background-position
- background-size

you could make a **css** class that contains these attributes and simple let every element also use this class, to reduce code duplication.